### ISSUE TITLE
Update action to use updated version of Node

### ISSUE THREAD

**Issue by jjoynson-iw** [denoland/setup-deno#35]
Seeing the following warning message when using this action:
```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: denoland/setup-deno
```

Can see node12 is set here: https://github.com/denoland/setup-deno/blob/main/action.yml#L17

**Comment by soundstep** [comment]
NodeJS 12 is [not supported anymore](https://github.com/nodejs/release#release-schedule), could we move setup-demo to Node 14?

**Comment by soundstep** [comment]
@jjoynson-iw Just noticed: https://github.com/denoland/setup-deno/pull/33

**Comment by lucacasonato** [comment]
Fixed in v1.1.1
